--- grep-2.5.3/configure.ac.orig	2007-06-30 13:35:20.000000000 +0200
+++ grep-2.5.3/configure.ac	2007-08-28 19:34:17.219123302 +0200
@@ -96,7 +96,7 @@
 esac
 
 dnl I18N feature
-ALL_LINGUAS="af be bg ca cs da de el eo es et eu fi fr ga gl he hr hu id it ja ko nb nl no pl pt_BR pt ro ru rw sl sr sv tr uk vi zh_TW"
+ALL_LINGUAS="af be bg ca cs da de el eo es et eu fi fr ga gl he hr hu id it ja ko nb nl pl pt_BR pt ro ru rw sl sr sv tr uk vi zh_TW"
 AM_GNU_GETTEXT
 
 dnl DOS file name convention
