--- grep-2.5.3/lib/error.c.orig	2007-06-28 20:57:18.000000000 +0200
+++ grep-2.5.3/lib/error.c	2007-08-28 19:41:36.200139404 +0200
@@ -249,7 +249,7 @@
     {
 #if defined HAVE_STRERROR_R || _LIBC
       char errbuf[1024];
-# if HAVE_WORKING_STRERROR_R || _LIBC
+# if STRERROR_R_CHAR_P || _LIBC
       fprintf (stderr, ": %s", __strerror_r (errnum, errbuf, sizeof errbuf));
 # else
       /* Don't use __strerror_r's return value because on some systems
